gallery/src/components/NxFormSelect/NxFormSelectPage.tsx

```diff
@@ -109,6 +111,32 @@ const NxFormSelectPage = () =>
             </NxTable.Row>
           </NxTable.Body>
         </NxTable>
+        <NxH3>NxFormSelect CSS Classes</NxH3>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Class</NxTable.Cell>
+              <NxTable.Cell>Location</NxTable.Cell>
+              <NxTable.Cell>Details</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-form-select--short</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Directly on the <NxCode>NxFormSelect</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>Short variant (150px) of <NxCode>NxFormSelect</NxCode>.</NxTable.Cell>
```

Comment:
I'd recommend against putting the numbers in the descriptions as they could easily get out of date

Reply:
> I'd recommend against putting the numbers in the descriptions as they could easily get out of date

Fair: https://github.com/sonatype/sonatype-react-shared-components/pull/586/commits/a5747a23a7dd4b7066d89bae5ad65e3239677c50